mapper/mapper.go

```diff
@@ -80,6 +81,19 @@ func ReleaseCalendar(rawReleaseCalendar release_calendar.ReleaseCalendar) *model
 	return &rc
 }
 
+// FeaturedContent takes the homepageContent as returned from the client and returns an array of featured content
+func FeaturedContent(homepageData zebedee.HomepageContent) []model.Feature {
+	var mappedFeaturesArray []model.Feature
```

Comment:
Would `mappedFeatureContent` (or similar) be better than `mappedFeaturesArray` ?